web/gds-admin-ui/src/pages/app/dashboard/Status.js

```diff
@@ -1,16 +1,35 @@
-// @flow
 import React from 'react';
 import { Doughnut } from 'react-chartjs-2';
 import { Card, Row, Col } from 'react-bootstrap';
+import { capitalizeFirstLetter, convertCountsToPercentages } from '../../../utils';
 
-const Status = (): React$Element<any> => {
+const Status = ({ statuses }) => {
     const colors = ['#0acf97', '#727cf5', '#fa5c7c'];
 
+    const getDonutChartData = () => {
+        if (statuses && typeof statuses === "object") {
+            return Object.values(convertCountsToPercentages(statuses))
+        }
+
+        return []
+    }
+
+    const getDonutChartLabels = () => {
+        if (statuses && typeof statuses === "object") {
+            return Object.keys(statuses).map(status => {
+                const status_ = capitalizeFirstLetter(status).split('_')
+                return status_.join(' ')
+            })
+        }
+
+        return []
+    }
```

Comment:
I don't think this will be a problem - but we may want to check this with a test in the future (please create a story); the order of the data array needs to match the order of the labels array.